### PR TITLE
Code-review.1.0

### PR DESCRIPTION
**Pull request by vish-devs**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: 'Your name here'
```
Comment (deepan) [inline action.yml]: Do we need this file ?

File: action.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  CONFIG_PATH:
+    required: true
+    description: 'Path to config file used in action'
+  TARGET_BRANCH:
+    required: true
+    description: 'Target branch of PR'
+  
+runs:
+  using: 'node16'
+  main: 'dist/index.js'
```
Comment (deepan) [inline action.yml:26]: Don't checkin dist files to git repo

File: src/fileHandler.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+export class FrozenBranches {
```
Comment (deepan) [inline src/fileHandler.ts]: Follow correct naming conventions. Also match the file name to the class name

File: src/main.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  if (event_type !== 'push') baseBranch = targetBranch
+  core.info(` The base branch is ${baseBranch}`)
+
+  let frozenBranches
+  let isBranchPresent
+
+  try {
+    const frozenBranchData = await getS3Object(bucketName, s3FileKey)
+    if (!frozenBranchData) {
+      core.info(
```
Comment (deepan) [inline src/main.ts]: Remove the log and simplify the if condition (remove else)

**Review by deepan — CHANGES_REQUESTED** [review]

File: action.yml (hunk 1 above)
Comment (vish-devs) [reply action.yml]: Yes. That will be that entry point for this action.

File: action.yml (hunk 2 above)
Comment (vish-devs) [reply action.yml:26]: The compiled files are in lib directory. And we are not checkin the lib directory on github.
We will have to have the dist directory for the action to run.
Is it okay?

File: src/main.ts (hunk 4 above)
Comment (vish-devs) [reply src/main.ts]: Done.

**Review by deepan — APPROVED** [review]

**Pull request merged**